packages/@ama-sdk/core/src/plugins/core/request-plugin.ts

```diff
@@ -52,11 +53,19 @@ export interface RequestOptions extends RequestInit {
   method: NonNullable<RequestInit['method']>;
 }
 
+/**
+ * Interface of an SDK request plugin context.
+ */
+export interface RequestPluginContext {
+  /** Logger (optional, fallback to console logger if undefined) */
+  logger?: Logger;
+}
+
 /**
  * Interface of an SDK request plugin.
  * The plugin will be run on the request of a call
  */
 export interface RequestPlugin extends Plugin<RequestOptions, RequestOptions> {
   /** Load the plugin with the context */
-  load(): PluginRunner<RequestOptions, RequestOptions>;
+  load(context?: RequestPluginContext): PluginRunner<RequestOptions, RequestOptions>;
```

Comment:
JSDoc

Reply:
JSDoc is above

Reply:
I think he is referring the parameter.
You may be able also to replace the comment by `/** inheritDoc */`

Reply:
ah I was using the same format as the other core plugins that also have parameters, should I update them all?

Reply:
ah I was using the same format as the other core plugins that also have parameters, should I update them all?

Reply:
if `/** inheritDoc */` do the job, I would say "yes". If not, only adding the missing parameter is ok